modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/IgniteTableImpl.java

```diff
@@ -174,103 +139,43 @@ public IgniteDistribution distribution() {
 
     /** {@inheritDoc} */
     @Override
-    public Map<String, IgniteIndex> indexes() {
-        return Collections.unmodifiableMap(indexes);
+    public Schema.TableType getJdbcTableType() {
+        return Schema.TableType.TABLE;
     }
 
     /** {@inheritDoc} */
     @Override
-    public void addIndex(IgniteIndex idxTbl) {
-        indexes.put(idxTbl.name(), idxTbl);
+    public boolean isRolledUp(String column) {
+        return false;
     }
 
     /** {@inheritDoc} */
     @Override
-    public IgniteIndex getIndex(String idxName) {
-        return indexes.get(idxName);
+    public boolean rolledUpColumnValidInsideAgg(String column, SqlCall call,
+            @Nullable SqlNode parent,
+            @Nullable CalciteConnectionConfig config) {
+        return false;
     }
 
     /** {@inheritDoc} */
     @Override
-    public void removeIndex(String idxName) {
-        indexes.remove(idxName);
+    public Statistic getStatistic() {
+        return statistic;
     }
 
     /** {@inheritDoc} */
     @Override
-    public <C> C unwrap(Class<C> cls) {
+    public <C> @Nullable C unwrap(Class<C> cls) {
         if (cls.isInstance(desc)) {
             return cls.cast(desc);
         }
 
         return super.unwrap(cls);
     }
 
-    static DoubleSupplier rowCountStatistic(InternalTable table) {
-        return new RowCountStatistic(table);
-    }
-
-    private static final class RowCountStatistic implements DoubleSupplier {
```

Comment:
I can`t found new statistic implementation (
Also you remove informational issue // TODO: need to be refactored https://issues.apache.org/jira/browse/IGNITE-19558 we need to remember about it )